### PR TITLE
fix(tracking): remove URI params on product label

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-11027

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-11027
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/error/error.routing.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ export default /* @ngInject */ ($stateProvider) => {
         const [, product] = $state
           .href(state, params, { lossy: true })
           .split('/');
-        return product;
+        // URI params are removed from error tracking
+        return product ? product.split('?')[0] : product;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/error/error.routing.js:24]:
```suggestion
        return product?.split('?')[0];
```

File: packages/manager/apps/dedicated/client/app/expired/expired.routing.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href('app.configuration', { reload: true }),
       error: /* @ngInject */ ($transition$) =>
         get($transition$.params(), 'error'),
-      product: /* @ngInject */ ($transition$) => $transition$.params()?.product,
+      product: /* @ngInject */ ($transition$) => {
+        // URI params are removed from error tracking
+        const product = $transition$.params()?.product;
+        return product ? product.split('?')[0] : product;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/expired/expired.routing.js:23]:
```suggestion
        return product?.split('?')[0];
```

File: packages/manager/apps/telecom/src/app/error-page/error-page.routing.js [hunk 3]
```diff
@@ -17,7 +17,8 @@ export default /* @ngInject */ ($stateProvider) => {
         const [, product] = $state
           .href(state, params, { lossy: true })
           .split('/');
-        return product;
+        // URI params are removed from error tracking
+        return product ? product.split('?')[0] : product;
```
Comment (anooparveti) [inline packages/manager/apps/telecom/src/app/error-page/error-page.routing.js:21]:
```suggestion
       return product?.split('?')[0];
```

File: packages/manager/apps/web/client/app/error-page/error-page.routing.js [hunk 4]
```diff
@@ -17,7 +17,8 @@ export default /* @ngInject */ ($stateProvider) => {
         const [, product] = $state
           .href(state, params, { lossy: true })
           .split('/');
-        return product;
+        // URI params are removed from error tracking
+        return product ? product.split('?')[0] : product;
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/error-page/error-page.routing.js:21]:
```suggestion
         return product?.split('?')[0];
```

Comment (darsene) [reply packages/manager/apps/web/client/app/error-page/error-page.routing.js]: Fixed!

File: packages/manager/apps/telecom/src/app/error-page/error-page.routing.js (hunk 3 above)
Comment (darsene) [reply packages/manager/apps/telecom/src/app/error-page/error-page.routing.js]: Fixed!

Comment (darsene) [reply packages/manager/apps/telecom/src/app/error-page/error-page.routing.js]: Fixed!

File: packages/manager/apps/dedicated/client/app/expired/expired.routing.js (hunk 2 above)
Comment (darsene) [reply packages/manager/apps/dedicated/client/app/expired/expired.routing.js]: Fixed!

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**